src/inmanta/data/__init__.py

```diff
@@ -3609,7 +3609,7 @@ async def get_next_compiles_for_environment(cls, environment_id: uuid.UUID) -> "
     @classmethod
     async def get_next_compiles_count(cls) -> int:
         """Get the number of compiles in the queue for ALL environments"""
-        result = await cls._fetch_int(f"SELECT count(*) FROM {cls.table_name()} WHERE NOT handled and completed IS NOT NULL")
+        result = await cls._fetch_int(f"SELECT count(*) FROM {cls.table_name()} WHERE NOT handled AND completed IS NULL")
```

Comment:
This looks like a pretty big bug. How comes this didn't bite use before?

Reply:
only used in the status page,....